src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -326,7 +326,6 @@ export class DashboardListing extends React.Component {
         sortable: true,
         render: (field, record) => (
           <EuiLink
-            className="dashboardLink"
```

Comment:
@cchaos The dashboard_listing function tests are failing because this class was removed. This class is used by [getCountOfDashboardsInListingTable](https://github.com/elastic/kibana/blob/master/test/functional/page_objects/dashboard_page.js#L381) to find all dashboard names listed in the table.

Reply:
@nreese Ahhhh ok, thanks! I'll get that fixed.